.github/workflows/build.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  pull_request:
+    branches: [ "main" ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Go ${{ matrix.go }} sample
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.15'
+      - run: go test ./...
+      - run: go build -o firebase-ctl ./cmd/firebase-ctl
```

Comment:
when we have make files can we not use those ?

Reply:
Using make now